firestore/tests/unit/v1/test_client.py

```diff
@@ -84,6 +95,26 @@ def test__firestore_api_property(self, mock_client):
         self.assertIs(client._firestore_api, mock_client.return_value)
         self.assertEqual(mock_client.call_count, 1)
 
+    @mock.patch(
+        "google.cloud.firestore_v1.gapic.firestore_client." "FirestoreClient",
+        autospec=True,
+        return_value=mock.sentinel.firestore_api,
+    )
+    def test__firestore_api_property_with_emulator(self, mock_client):
+        emulator_host = "localhost:8081"
+        with mock.patch("os.getenv") as getenv:
+            getenv.return_value = emulator_host
+            client = self._make_default_one()
+
+        self.assertIsNone(client._firestore_api_internal)
+        firestore_api = client._firestore_api
+        self.assertIs(firestore_api, mock_client.return_value)
+        self.assertIs(firestore_api, client._firestore_api_internal)
```

Comment:
Needs assertion(s) that an insecure channel was constructed with the expected emulator URL.